ts/Core/Axis/Color/ColorAxis.ts

```diff
@@ -897,6 +897,14 @@ class ColorAxis extends Axis implements AxisLike {
         const valueDecimals = (legendOptions as any).valueDecimals;
         const valueSuffix = (legendOptions as any).valueSuffix || '';
 
+        const getPointsInDataClass = (i: number): Point[] =>
```

Comment:
Use `Array<Point>` so that it is consistent and easy to see whether something is an array or a tuple.